### PR TITLE
rm default ui providers

### PR DESCRIPTION
**Pull request by aruniverse**

### CODE REVIEW

File: packages/apps/web-viewer-test/src/components/home/ViewerHome.tsx [hunk 1]
```diff
@@ -80,6 +98,14 @@ export const ViewerHome: React.FC = () => {
           },
         }}
         enablePerformanceMonitors={true}
+        onIModelAppInit={onIModelAppInit}
+        uiProviders={[
+          new TreeWidgetUiItemsProvider(),
+          // new PropertyGridUiItemsProvider({
```
Comment (kckst8) [inline packages/apps/web-viewer-test/src/components/home/ViewerHome.tsx:104]: waiting on something here?

File: packages/apps/web-viewer-test/tsconfig.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx",
+    "jsx": "react",
```
Comment (kckst8) [inline packages/apps/web-viewer-test/tsconfig.json:20]: I think we should leave this (for React 17). Any reason to the contrary?

**Review by kckst8 — APPROVED** [review]
few minor questions/comments. Looks good otherwise. Finally, you get your wish :-)

File: packages/apps/web-viewer-test/src/components/home/ViewerHome.tsx (hunk 1 above)
Comment (aruniverse) [reply packages/apps/web-viewer-test/src/components/home/ViewerHome.tsx:104]: https://github.com/iTwin/viewer-components-react/pull/185

Cant init PropertyGrid yet

**Comment by aruniverse** [comment]
> Finally, you get your wish :-)

😁 Let me know if you think this is ok to go into `next` or if we should defer it

**Pull request merged**